connector/pom.xml

```diff
@@ -153,9 +159,37 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-resolver-dns</artifactId>
-            <version>4.1.43.Final</version>
+            <version>4.1.56.Final</version>
```

Comment:
Might as well bump this up to 4.1.58-Final? In my dependency PR, I added a Maven property for the Netty version; feel free to do that here.